enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -374,3 +349,41 @@ Provide a recipe for customer's to deploy their own log collector to move the re
 
 ## Infrastructure Needed
 * Future target endpoints may require special infrastructure or licencing to properly test.  For example, Splunk is destination requested by many users but is a licensed product.  The free dev tier may not provide sufficient capacity to fully test
+* Scale and stress tests require intensive use of a large cluster for an extended period of time.
+## Future plans
+
+As well as serving the current GA requirements, the log forwarding API has been designed with the following future requirements in mind.
+
+### Stand-alone log forwarding. ###
+
+Deploy log forwarding without deploying the entirety of the cluster logging infrastructure (e.g. Kibana, Elasticsearch) Forwarding will be a stand-alone system independent of any log store. This decoupling will let us test forwarding separately, and let customers to switch off our managed store entirely while still using a managed and supported forwarder.
```

Comment:
i thought we were enabling this in the initial GA.  If i define an alternate forward (and do not include the default forward), am i still going to get kibana+elastic deployments?

Reply:
the deployments are controlled external to the log forward, log forward is just about controlling the flow of logs.

Reply:
> the deployments are controlled external to the log forward, log forward is just about controlling the flow of logs.

right, but i thought we said that unless there is a log forward config that points to our internal elastic, we would not deploy our internal elastic(+kibana).

Reply:
So the story is:
- if you have no log forwarding config at all, we default to existing elastic.
- if you have a log forwarding config you must explicitly enable the default stuff using the built-in output name "Default"